features/support/helpers.rb

```diff
@@ -74,6 +74,10 @@ def click_save_and_continue
   end
 end
 
+def submit
+  find("button.govuk-button:not(.govuk-button--secondary)").click
```

Comment:
This is a good plan - but I'm trying to lean away from using CSS classes when looking for buttons / links and leaning more towards data attributes, so if we give the button a `data-testid` of `submit-button` (or whatever), we can more reliably find the button, and not have to worry about changes in CSS or text breaking tests